docs/SOPs/contributor_communication_SOP.md

```diff
@@ -214,15 +214,7 @@ Note that this is a constantly changing space, this email was last updated 2020-
 > 
 > 2 Upload your data to our cloud space
 > 
-> To upload your raw data files to our cloud space you will need to follow the instructions here: [How to upload data to an hca upload area with hca-util] (https://github.com/ebi-ait/hca-documentation/wiki/How-to-upload-data-to-an-upload-area-using-hca-util) 
-> 
-> The credentials you will need are:
-> 
-> Unique upload area identifier: <upload area uuid>
-> 
-> AWS Access key: <contributor access key> 
-> 
-> AWS Secret key: <contributor secret key>
+> The credentials you will need will be sent separately by a developer in our team.
```

Comment:
not necessarily a developer. Ideally, this would be automated.
```suggestion
> The credentials you will need will be sent separately by a member of the inges team.
```

Reply:
```suggestion
> The credentials you will need will be sent in a separate email.
```
Actually, they don't need to know the role of the person that will send the email at all